lib/array.js

```diff
@@ -198,4 +198,33 @@ module.exports = (api) => {
     else done();
   };
 
+  api.metasync.every = (
+    // Asynchronous every
+    items, //incoming array
+    fn, // function(value, callback)
+    //   value - item from items array
+    //   callback - function(err, fits)
+    //     err - error or null
+    //     fits - true/false returned from fn
+    done // optional on done callback function(err, result)
+    //   err - error or null
+    //   result - true/false, does every element of array fit predicate
+  ) => {
+    done = api.metasync.cb(done);
+
+    let proceedItemsCount = 0;
+
+    function handleFnResult(err, accepted) {
+      if (err) return done(err);
+      if (!accepted) return done(null, false);
+
+      proceedItemsCount++;
+      if (proceedItemsCount === items.length) done(null, true);
+    }
+
+    for (const item of items) {
```

Comment:
@tshemsedinov Do you mean `let item; for (item of items)`?
Also maybe it's good case for using `for (var item of items)`? What is difference?